### ISSUE TITLE
Addition to Curator Stats 

### ISSUE THREAD

**Issue by dazzariti** [clingen-data-model/clinvar-scv-curator-ext#46]
After discussion with Heidi in our Nov 22 meeting, she liked to be able to see a count  unique SCVs per Reason per CURATOR for Most Recent Non-Contributory Non-Retired Action.

This would be very similar to the "Non-Contributor Stats1" tab but would be by curator rather than submitter.

Not urgent! Just want to make sure I have this documented.

**Comment by dazzariti** [comment]
Reviewed on January 13 CvC Call, this is still not an urgent priority, but it is a request we will need when we start bringing on new curators to train on CvC curation

**Comment by larrybabb** [comment]
@dazzariti I'm moving this to "Selected for Development" for now. If you think we are a ways out from bringing on new curators to train then feel free to stick it in the "Backlog". I defer to you to let me know when this should make it to the active work list.

**Comment by dazzariti** [comment]
Do you have an estimate for the amount of work? If it's going to take two weeks, it's probably not a priority.  If it's something you can do in a day I think it would be helpful to have available. Thanks!

**Comment by larrybabb** [comment]
less than a day.

**Comment by dazzariti** [comment]
I agree with Selected for Development then, thanks!

**Comment by larrybabb** [comment]
this was completed a while ago 
https://docs.google.com/spreadsheets/d/1DmYRW7YYmLnsYuQNkHEMAOx-d6mdLD5lCTky0p1UDbw/edit#gid=200073685
@dazzariti can you verify that the #4 report on the "SCV Curation Statistics" tab of the google sheet above is aligned with this old request? If it isn't can you clarify what else would be needed?

**Comment by dazzariti** [comment]
@larrybabb  I don't believe this is complete, the request was to break out the Non Contributory reasons per curator, which I don't see in that sheet 

It's essentially a combination of #4 and #5
